frontends/web/src/routes/account/send/send.tsx

```diff
@@ -527,14 +527,16 @@ class Send extends Component<Props, State> {
                 ) }
                 <div className={`flex flex-row flex-between ${style.container}`}>
                   <label className="labelXLarge">{t('send.transactionDetails')}</label>
-                  { coinControl && (
-                    <Button
-                      className="m-bottom-quarter p-right-none"
-                      transparent
-                      onClick={this.toggleCoinControl}>
-                      {t('send.toggleCoinControl')}
-                    </Button>
-                  )}
+                  <div className={style.coinControlButtonContainer}>
```

Comment:
I haven't tested but shouldn't that div element be inside the condition in case a user doesn't have coin-control enabled?